libs/coin-evm/src/__tests__/unit/api/explorer/index.unit.test.ts

```diff
@@ -31,6 +33,14 @@ describe("EVM Family", () => {
         expect(explorerA).toBe(etherscanLikeApi);
         expect(explorerB).toBe(etherscanLikeApi);
       });
+
+      it("should return the ledger api", async () => {
+        const explorerA = await getExplorerApi({
+          ethereumLikeInfo: { explorer: { type: "ledger", explorerId: "eth" } },
+        } as any);
```

Comment:
Yes & no, both are bad since this is definitely not a CryptoCurrency as we're missing 90% of the object. I don't think we should care about using `any` in tests.